app/ide-desktop/lib/types/modules.d.ts

```diff
@@ -14,9 +15,35 @@ declare module '*/build.json' {
     export default BUILD_INFO
 }
 
+// Required because this is a build artifact and so would not otherwise work on a clean repository.
 declare module '*/ensogl-pack/linked-dist' {
-    // eslint-disable-next-line no-restricted-syntax
-    export * from '../../../../lib/rust/ensogl/pack/js/src/runner/index'
+    /* eslint-disable @typescript-eslint/consistent-type-imports, no-restricted-syntax, @typescript-eslint/no-shadow */
+    export type App = import('runner/index').App
+    export const App: typeof import('runner/index').App
+    export type Consumer = import('runner/index').Consumer
+    export const Consumer: typeof import('runner/index').Consumer
+    export type LogLevel = import('runner/index').LogLevel
+    export const LogLevel: typeof import('runner/index').LogLevel
+    export type Logger = import('runner/index').Logger
+    export const Logger: typeof import('runner/index').Logger
+    export type Option<T> = import('runner/index').Option<T>
+    export const Option: typeof import('runner/index').Option
+    export const log: typeof import('runner/index').log
+    export const logger: typeof import('runner/index').logger
+    export const urlParams: typeof import('runner/index').urlParams
+    export namespace config {
+        export type Option<T> = import('runner/index').config.Option<T>
+        export const Option: typeof import('runner/index').config.Option
+        export type Group<
+            Options extends OptionsRecord,
+            Groups extends GroupsRecord
+        > = typeof import('runner/index').config.Group<Options, Groups>
+        export const Group: typeof import('runner/index').config.Group
+        export const objectToGroup: typeof import('runner/index').config.objectToGroup
+        export const objectToOption: typeof import('runner/index').config.objectToOption
```

Comment:
this is terribly verbose and hard to maintain. I understand that the second PR is an alternative to this one, without this verbosity. Am I correct?

Reply:
yeah - i definitely agree the other one is cleaner... and more importantly, doesnt require manually adding or removing exports